physics/kepler_orbit.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "physics/body.hpp"
+#include "physics/degrees_of_freedom.hpp"
+
+namespace principia {
+namespace physics {
+
+template<typename Frame>
+struct KeplerianElements {
+  double eccentricity;
+  Length semimajor_axis;
+  Angle inclination;
+  Angle longitude_of_ascending_node;
+  Angle argument_of_periapsis;
+  Angle mean_anomaly;
+};
+
+struct TwoBodySystem {
```

Comment:
This struct doesn't seem too useful.  I would prefer to pass the two bodies at construction.